KPI/H.adoc

```diff
@@ -1,99 +1,239 @@
-== H. Data Parameter Keyword
+== H. Keywords
 
-Data parameter keywords are optional elements syntactically.  
-An xml record will validate without its presence.
+=== WMCP element(s)
 
+....
+//gmd:MD_DataIdentification/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:keyword
+//gmd:MD_DataIdentification/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:type
+//gmd:MD_DataIdentification/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:thesaurusName
+....
 
-=== H.1 To which WMCP element(s) does it apply ?
+=== What is being measured
 
-This KPI applies to //<gmd:identificationInfo>/*/gmd:descriptiveKeywords/*/gmd:keyword/
-WMO Core Profile extends the CodeLists for MD_keywordTypeCode code to include "dataParam" and "dataCentre" values. 
+* presence of keywords, the level of definition and reference to controlled
+vocabularies
+* level of definition of a keyword will improve the searchability of a dataset
 
-To express Parameter keywords, one shall specify:
+In addition to measure well defined references to dictionaries and keyword
+types, compliance to WMCP 1.3 defines a set of rules that apply to different
+sets of keywords.  These have been listed for reference but are implemented
+as part of KPI A (compliance).
 
-1 Keyword: These can be expressed via CharacteString, or if reference is needed to a specific online resource/code an anchor could be introduced.
-See examples with Character String vs anchors:https://webgate.ec.europa.eu/fpfis/wikis/display/InspireMIG/Spatial+scope+code+list
+=== Rationale for measurement
 
-2 type:  dataParam (MD_KeywordTypeCode)
- /gmd:MD_Metadata/gmd:identificationInfo/*/gmd:descriptiveKeywords/*/gmd:type/*/@codeListValue=”dataParam”
+Encouraging metadata providers to make use of keywords that are published in
+controlled vocabularies will ultimately help the end user to search for
+well-known domain related terms.
 
-3 thesaurusName: Originating controlled vocabulary. 
-https://www.wmo.int/pages/prog/www/WIS/Metadata/WMOCoreMetadataVer1.html (suggest to add anchors for codelists)
+Keywords are indexed by search engines to narrow down full text searches,
+adding to the user experience and making datasets easier to discover. Keywords
+can be user-defined or specified from controlled vocabularies.
 
+=== Rules for implementation
 
-Restrictions:
+==== Compliance rules 
 
-8.2.3 All keywords sourced from a particular keyword thesaurus shall be grouped into a
-single instance of the MD_Keywords class.
+As described in https://github.com/wmo-im/wmcp/blob/master/metadata/WMO_Core_Metadata_Profile_v1.3_Part_2.pdf[WMCP 1.3, Part 2]:
```

Comment:
Line 36 Link needs updating to:
https://github.com/wmo-im/wcmp/blob/master/specification/WMO_Core_Metadata_Profile_v1.3_Part_2.pdf

Reply:
Good catch @martaguti, updated.